index.html

```diff
@@ -707,14 +707,13 @@ <h2 class="page-section-heading text-center text-uppercase text-secondary mb-3">
                                     href="https://tigerbook.herokuapp.com/"
                                     target="_blank"
                                     class="no-underline"
-                                    >TigerBook</a
+                                    >Tigerbook</a
                                 >
                             </p>
                             <p class="fs-6 text-center fst-italic text-secondary mb-0">
-                                A unified directory of Princeton undergraduates. By Hansen Qian '16,
-                                Ivo Crnkovic-Rubsamen '15, Rohan Sharma '14, and Adam Libresco '19.
+                                A directory of Princeton undergraduate students.
                                 <br />
-                                Note: TigerBook is not managed by TigerApps.
+                                Note: Tigerbook has been retired in favor of the official Princeton directory.
```

Comment:
"The original student-built Tigerbook has been retired in favor of the Registrar's official Princeton directory". The original wording is a bit confusing cuz we're calling this new directory "Tigerbook" while saying that it's retired?